BUILD.bazel

```diff
@@ -1,5 +1,48 @@
 load("@io_bazel_rules_go//go:def.bzl", "go_library")
 load("@bazel_gazelle//:def.bzl", "gazelle")
+load("@io_bazel_rules_go//go:def.bzl", "nogo")
+load("@com_github_atlassian_bazel_tools//goimports:def.bzl", "goimports")
+
+goimports(
```

Comment:
IIUC we still need `goimports` under the builder container, because we want to support pure go compilation.
In general, it can be great to remove all go packages from the builder container and apply them via bazel, but it should be separate PR.

Reply:
I would rather say, that we support plain go commands. Formatting and so on is not necessarily part of the deal, from my point of view, since any code change might potentially need e.g. BUILD.bazel file upates anyway.